.storybook/stories/datagrid/datagrid.stories.ts

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2016-2022 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { ClrCommonStringsService, ClrConditionalModule, ClrDatagrid, ClrDatagridModule } from '@clr/angular';
+import { action } from '@storybook/addon-actions';
+import { Parameters } from '@storybook/addons';
+import { Story } from '@storybook/angular';
+
+import { elements } from '../../helpers/elements.data';
+import { setupStorybook } from '../../helpers/setup-storybook.helpers';
+
+const defaultStory: Story = args => ({
+  template: `
+  <style>.electronegativity-container { border-bottom: 4px solid #119cd4; }</style>
```

Comment:
nitpick: This template isn't indented like the others.

Reply:
fixed